2024-03-19 14:58:17,081 INFO    MainThread:10260 [wandb_setup.py:_flush():76] Current SDK version is 0.16.3
2024-03-19 14:58:17,082 INFO    MainThread:10260 [wandb_setup.py:_flush():76] Configure stats pid to 10260
2024-03-19 14:58:17,082 INFO    MainThread:10260 [wandb_setup.py:_flush():76] Loading settings from C:\Users\CYP\.config\wandb\settings
2024-03-19 14:58:17,082 INFO    MainThread:10260 [wandb_setup.py:_flush():76] Loading settings from c:\Users\CYP\Documents\GitHub\AIAKI\wandb\settings
2024-03-19 14:58:17,082 INFO    MainThread:10260 [wandb_setup.py:_flush():76] Loading settings from environment variables: {}
2024-03-19 14:58:17,082 INFO    MainThread:10260 [wandb_setup.py:_flush():76] Applying setup settings: {'_disable_service': False}
2024-03-19 14:58:17,082 INFO    MainThread:10260 [wandb_setup.py:_flush():76] Inferring run settings from compute environment: {'program': '<python with no main file>'}
2024-03-19 14:58:17,082 INFO    MainThread:10260 [wandb_init.py:_log_setup():526] Logging user logs to c:\Users\CYP\Documents\GitHub\AIAKI\wandb\run-20240319_145817-ol78oovf\logs\debug.log
2024-03-19 14:58:17,082 INFO    MainThread:10260 [wandb_init.py:_log_setup():527] Logging internal logs to c:\Users\CYP\Documents\GitHub\AIAKI\wandb\run-20240319_145817-ol78oovf\logs\debug-internal.log
2024-03-19 14:58:17,082 INFO    MainThread:10260 [wandb_init.py:_jupyter_setup():472] configuring jupyter hooks <wandb.sdk.wandb_init._WandbInit object at 0x00000213D3F00220>
2024-03-19 14:58:17,083 INFO    MainThread:10260 [wandb_init.py:init():566] calling init triggers
2024-03-19 14:58:17,083 INFO    MainThread:10260 [wandb_init.py:init():573] wandb.init called with sweep_config: {}
config: {'batch_size': 64, 'n_units': 8, 'learning_rate': 0.001, 'lr_decay': 1, 'lr_step': 5, 'ns': 6, 'loss': 'mse'}
2024-03-19 14:58:17,083 INFO    MainThread:10260 [wandb_init.py:init():616] starting backend
2024-03-19 14:58:17,083 INFO    MainThread:10260 [wandb_init.py:init():620] setting up manager
2024-03-19 14:58:17,084 INFO    MainThread:10260 [backend.py:_multiprocessing_setup():105] multiprocessing start_methods=spawn, using: spawn
2024-03-19 14:58:17,085 INFO    MainThread:10260 [wandb_init.py:init():628] backend started and connected
2024-03-19 14:58:17,089 INFO    MainThread:10260 [wandb_run.py:_label_probe_notebook():1295] probe notebook
2024-03-19 14:58:17,090 INFO    MainThread:10260 [wandb_run.py:_label_probe_notebook():1305] Unable to probe notebook: 'NoneType' object has no attribute 'get'
2024-03-19 14:58:17,090 INFO    MainThread:10260 [wandb_init.py:init():720] updated telemetry
2024-03-19 14:58:17,113 INFO    MainThread:10260 [wandb_init.py:init():753] communicating run to backend with 90.0 second timeout
2024-03-19 14:58:17,639 INFO    MainThread:10260 [wandb_run.py:_on_init():2262] communicating current version
2024-03-19 14:58:17,666 INFO    MainThread:10260 [wandb_run.py:_on_init():2271] got version response upgrade_message: "wandb version 0.16.4 is available!  To upgrade, please run:\n $ pip install wandb --upgrade"

2024-03-19 14:58:17,666 INFO    MainThread:10260 [wandb_init.py:init():804] starting run threads in backend
2024-03-19 14:58:17,796 INFO    MainThread:10260 [wandb_run.py:_console_start():2241] atexit reg
2024-03-19 14:58:17,796 INFO    MainThread:10260 [wandb_run.py:_redirect():2096] redirect: wrap_raw
2024-03-19 14:58:17,796 INFO    MainThread:10260 [wandb_run.py:_redirect():2161] Wrapping output streams.
2024-03-19 14:58:17,796 INFO    MainThread:10260 [wandb_run.py:_redirect():2186] Redirects installed.
2024-03-19 14:58:17,797 INFO    MainThread:10260 [wandb_init.py:init():847] run started, returning control to user process
2024-03-19 15:28:37,803 INFO    MainThread:10260 [jupyter.py:save_ipynb():373] not saving jupyter notebook
2024-03-19 15:28:37,803 INFO    MainThread:10260 [wandb_init.py:_pause_backend():437] pausing backend
2024-03-19 15:28:39,135 INFO    MainThread:10260 [wandb_init.py:_resume_backend():442] resuming backend
2024-03-19 15:28:39,143 INFO    MainThread:10260 [wandb_setup.py:_flush():76] Current SDK version is 0.16.3
2024-03-19 15:28:39,143 INFO    MainThread:10260 [wandb_setup.py:_flush():76] Configure stats pid to 10260
2024-03-19 15:28:39,143 INFO    MainThread:10260 [wandb_setup.py:_flush():76] Loading settings from C:\Users\CYP\.config\wandb\settings
2024-03-19 15:28:39,143 INFO    MainThread:10260 [wandb_setup.py:_flush():76] Loading settings from c:\Users\CYP\Documents\GitHub\AIAKI\wandb\settings
2024-03-19 15:28:39,143 INFO    MainThread:10260 [wandb_setup.py:_flush():76] Loading settings from environment variables: {}
2024-03-19 15:28:39,143 INFO    MainThread:10260 [wandb_setup.py:_flush():76] Applying setup settings: {'_disable_service': False}
2024-03-19 15:28:39,143 INFO    MainThread:10260 [wandb_setup.py:_flush():76] Inferring run settings from compute environment: {'program': '<python with no main file>'}
2024-03-19 15:28:39,143 INFO    MainThread:10260 [wandb_init.py:_log_setup():526] Logging user logs to c:\Users\CYP\Documents\GitHub\AIAKI\wandb\run-20240319_152839-6f77mhb8\logs\debug.log
2024-03-19 15:28:39,144 INFO    MainThread:10260 [wandb_init.py:_log_setup():527] Logging internal logs to c:\Users\CYP\Documents\GitHub\AIAKI\wandb\run-20240319_152839-6f77mhb8\logs\debug-internal.log
2024-03-19 15:28:39,144 INFO    MainThread:10260 [wandb_init.py:init():566] calling init triggers
2024-03-19 15:28:39,144 INFO    MainThread:10260 [wandb_init.py:init():573] wandb.init called with sweep_config: {}
config: {'batch_size': 64, 'n_units': 2, 'learning_rate': 0.0001, 'lr_decay': 0.9, 'lr_step': 2, 'ns': 4, 'loss': 'mse'}
2024-03-19 15:28:39,144 INFO    MainThread:10260 [wandb_init.py:init():591] re-initializing run, found existing run on stack: ol78oovf
2024-03-19 15:28:39,145 INFO    MainThread:10260 [wandb_run.py:_finish():1970] finishing run dahs/IMV_LSTM_AKI_new/ol78oovf
2024-03-19 15:28:39,145 ERROR   MainThread:10260 [jupyter.py:save_history():437] Run pip install nbformat to save notebook history
2024-03-19 15:28:39,145 INFO    MainThread:10260 [jupyter.py:save_ipynb():373] not saving jupyter notebook
2024-03-19 15:28:39,145 INFO    MainThread:10260 [wandb_init.py:_jupyter_teardown():454] cleaning up jupyter logic
2024-03-19 15:28:39,145 INFO    MainThread:10260 [wandb_run.py:_atexit_cleanup():2210] got exitcode: 0
2024-03-19 15:28:39,146 INFO    MainThread:10260 [wandb_run.py:_restore():2193] restore
2024-03-19 15:28:39,146 INFO    MainThread:10260 [wandb_run.py:_restore():2199] restore done
